addons/account/models/account_move.py

```diff
@@ -218,16 +218,22 @@ def assert_balanced(self):
         if not self.ids:
             return True
         prec = self.env['decimal.precision'].precision_get('Account')
+        prec = 10 ** (-max(5, prec))
```

Comment:
@nim-odoo Changes done
Not sure about the right way to write this line though...
Can I reassign the same variable or should I set a new one?

Reply:
Never mind ... Not necessary.